### PR TITLE
Added map web component shadow paths styling example

### PR DESCRIPTION
**Pull request by dwnoble**
Example of web component styling using shadow paths

Screenshot: Styled (top), unstyled (bottom)

styled-map

### CODE REVIEW

File: web-components/examples/example.html [hunk 1]
```diff
@@ -6,6 +6,36 @@
       href="https://datacommons.org/css/nl_interface.min.css"
     />
     <script src="https://datacommons.org/datacommons.js"></script>
+    <style>
```
Comment (juliawu) [inline web-components/examples/example.html:9]: Let's add a comment that this is some example CSS you can add to do custom styling of the components.

**Review by juliawu — APPROVED** [review]
This is super cool!

File: web-components/examples/example.html [hunk 2]
```diff
@@ -20,32 +80,32 @@ <h2>Bar Chart</h2>
     <datacommons-bar
       title="Population of select US states"
       variable="Count_Person"
-      comparisonPlaces="geoId/01 geoId/02 geoId/04 geoId/05"
+      comparisonPlaces='["geoId/01", "geoId/02", "geoId/04", "geoId/05"]'
```
Comment (juliawu) [inline web-components/examples/example.html:83]: Let's stick with space separated lists in the examples.

File: web-components/examples/example.html [hunk 3]
```diff
@@ -125,7 +209,23 @@ <h3>Line</h3>
...
+    <datacommons-line
+      title="Mean Palmer Drought Severity Index (PDSI) in the US"
+      place="country/USA"
+      variables='["Mean_PalmerDroughtSeverityIndex"]'
+    ></datacommons-line>
+    <datacommons-line
+      id="styled-line"
+      title="Styled: Mean Palmer Drought Severity Index (PDSI) in the US"
+      place="country/USA"
+      variables='["Mean_PalmerDroughtSeverityIndex"]'
```
Comment (juliawu) [inline web-components/examples/example.html:228]: Let's also go with space separated lists for `variables` in these examples.

**Review by juliawu — CHANGES_REQUESTED** [review]
Looks good! Let's just standardize space separated lists now that that change is in.

**Comment by dwnoble** [comment]
Thanks Julia- made those changes.

**Review by juliawu — APPROVED** [review]
Awesome! Thanks for all the changes!

**Pull request merged**